github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -42,14 +42,12 @@ presubmits:
         - "/bin/bash"
         - "-c"
         - >
-            apt update &&
-            apt install gettext-base -y &&
```

Comment:
We don't need gettext-base ?

Reply:
I think we have it on the new bootstrap image according this this https://github.com/kubevirt/project-infra/blob/master/images/bootstrap/Dockerfile